### DISCUSSION TITLE
Futurs plans pour Dashlord

### DISCUSSION THREAD

**Discussion by edelagnier** [SocialGouv/dashlord#79] [Ideas]
Bonjour Julien, 

Comme discuté, je viens donc de faire les PR pour permettre une utilisation hors du contexte des ministères eux-même. 
Je vais commencer plusieurs projets clients et ne pourrait pas contribuer davantage pour le moment mais je serais intéressée pour le faire à l'avenir selon les disponibilités. 
As tu des plans pour le projet ou pour d'autres dans la même veine ?

Au plaisir d'en discuter

**Comment by revolunet** [comment]
Salut Erica, merci pour les PRs qui seront bien utiles 🙏 

Il y a [un board avec quelques idées d'issues ici](https://github.com/orgs/SocialGouv/projects/13) mais rien de planifié.

Non je n'ai pas de plan bien précis la suite, juste quelques idées :
 - améliorer les propositions de remediation ([un essai ici](https://github.com/SocialGouv/dashlord-actions/pull/89))
 - ajouter des scanners comme [repolinter](https://github.com/todogroup/repolinter) ou d'autres outils sécu

petit ping à @Anna-Livia qui avait peut-être d'autres pistes
